cypress-tests/cypress/e2e/PaymentUtils/Novalnet.js

```diff
@@ -0,0 +1,193 @@
+const successfulNo3DSCardDetails = {
+    card_number: "4000000000001091",
+    card_exp_month: "12",
+    card_exp_year: "25",
+    card_holder_name: "Joseph Doe",
+    card_cvc: "123",
+  };
+
+const successfulThreeDSTestCardDetails = {
+  card_number: "4000000000001091",
+  card_exp_month: "12",
+  card_exp_year: "25",
+  card_holder_name: "Joseph Doe",
+  card_cvc: "123",
+  };
+  
```

Comment:
Since both the card numbers are same, I do not find the need to have 2 `const` serving same purpose. One can be removed, and assignments in objects below can be updated.